### ISSUE TITLE
simple features in the Library of Congress IIIF server

### ISSUE THREAD

**Issue by gregorycrane** [scaife-viewer/beyond-translation-site#99]
The LOC's IIIF browser let's you define a region of interest and then generate a link to that particular detail. Is that something Openseadragon can do?

https://www.loc.gov/resource/amedsaid1831.dw027/?sp=5&st=text&clip=765,645,1148,500&ciw=154&rot=0

![image](https://user-images.githubusercontent.com/5159577/206303834-5a24c0a0-9926-42eb-aac9-367c0994e085.png)

![image](https://user-images.githubusercontent.com/5159577/206303865-6ad4f7d6-02d9-4353-8449-5e89eeaa5186.png)

**Comment by jacobwegner** [comment]
@gregorycrane: Yes; it looks like their viewer is powered by OpenSeadragon, and this plugin provides the clipping functionality:

https://github.com/picturae/openseadragonselection

There may be more required to create a standalone page such as https://www.loc.gov/resource/amedsaid1831.dw027/?sp=5&st=text&clip=765,645,1148,500&ciw=154&rot=0 though.
